card.js

```diff
@@ -0,0 +1,27 @@
+export function createCharacterCard(datatvshow) {
+  const newCard = document.createElement("li");
+  ul.append(newCard);
+  newCard.classList.add("card-container");
```

Comment:
please look into the HTML which class the li element should get.

Reply:
Yes we already checked it, thx.